src/storage/src/render/multi_worker_persist_sink.rs

```diff
@@ -0,0 +1,1123 @@
+                            for description in data.drain(..) {
+                                if collection_id.is_user() {
+                                    trace!(
+                                        "persist_sink {collection_id}/{shard_id}: \
+                                            write_batches: \
+                                            new_description: {:?}, \
+                                            desired_frontier: {:?}, \
+                                            batch_descriptions_frontier: {:?}",
+                                        description,
+                                        desired_frontier,
+                                        batch_descriptions_frontier,
+                                    );
+                                }
+                                let existing = in_flight_batches.insert(
+                                    description.clone(),
+                                    cap.delayed(description.0.first().unwrap()),
+                                );
+                                assert!(
+                                    existing.is_none(),
+                                    "write_batches: sink {} got more than one \
+                                        batch for description {:?}, in-flight: {:?}",
+                                    collection_id,
+                                    description,
+                                    in_flight_batches
+                                );
+                            }
+
+                            continue;
+                        }
+                        Event::Progress(frontier) => {
+                            batch_descriptions_frontier = frontier;
+                        }
+                    }
+                }
+                Some(event) = desired_input.next_mut() => {
+                    match event {
+                        Event::Data(_cap, data) => {
+                            // Extract desired rows as positive contributions to `correction`.
+                            if collection_id.is_user() && !data.is_empty() {
+                                trace!(
+                                    "persist_sink {collection_id}/{shard_id}: \
+                                        updates: {:?}, \
+                                        in-flight-batches: {:?}, \
+                                        desired_frontier: {:?}, \
+                                        batch_descriptions_frontier: {:?}",
+                                    data,
+                                    in_flight_batches,
+                                    desired_frontier,
+                                    batch_descriptions_frontier,
+                                );
+                            }
+
+                            // TODO: come up with a better default batch size here
+                            // (100 was chosen arbitrarily), and avoid having to make a batch
```

Comment:
Where does the number 100 appear in the code? I don't see it

Reply:
oh sorry, this is an old comment